cluster-stamp.json

```diff
@@ -289,14 +262,59 @@
                 {
                     "type": "providers/roleAssignments",
                     "apiVersion": "2018-09-01-preview",
-                    "name": "[concat('Microsoft.Authorization/', guid(concat(resourceGroup().id), variables('readerRole')))]",
+                    "name": "[concat('Microsoft.Authorization/', guid(resourceGroup().id, 'mi-appgateway-frontend', variables('keyVaultSecretsUserRole')))]",
+                    "dependsOn": [
+                        "[resourceId('Microsoft.KeyVault/vaults', variables('keyVaultName'))]",
+                        "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', 'mi-appgateway-frontend')]"
+                    ],
+                    "comments": "Grant the Azure Application Gateway managed identity with key vault reader role permissions; this allows pulling frontend and backend certificates.",
+                    "properties": {
+                        "roleDefinitionId": "[variables('keyVaultSecretsUserRole')]",
+                        "principalId": "[reference(resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', 'mi-appgateway-frontend')).principalId]",
+                        "principalType": "ServicePrincipal"
+                    }
+                },
+                {
+                    "type": "providers/roleAssignments",
```

Comment:
Looks like "2020-10-01-preview" is available

Reply:
Yeah, I thought about going through and updating them all in all files, but I knew that was a bit out of scope -- as other ones could be updated as well and this isn't a freshness pass.  The only one I did update was one to help "stage" those firewall changes a bit better.  I'll add this to the backlog, since the new API version also has some additional params